composer.json

```diff
@@ -375,9 +375,6 @@
                 "CKeditor Plugin - using a combined patch to avoid conflict (https://www.drupal.org/node/2928318)": "https://www.drupal.org/files/issues/2020-10-28/media_entity_download-ckeditor_plugin-2928318-20.patch",
                 "3106062 - Track the usage of Media entity through media_entity_download route with entity_usage": "patches/media_entity_download-track_media_entity_usage-3106062-2.patch"
             },
-            "drupal/memcache": {
-                "Split large (> 1mb) items (https://www.drupal.org/node/435694)": "https://www.drupal.org/files/issues/2019-02-26/435694-118-d8-memcache-large-items.patch"
-            },
```

Comment:
Is this patch no longer needed on PHP8?

Reply:
Its already in the latest version of the module. Which I've updated